code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -0,0 +1,199 @@
+/datum/eldritch_knowledge/base_void
+	name = "Glimmer of Winter"
+	desc = "Opens up the path of void to you. Allows you to transmute a knife in a sub-zero temperature into a void blade."
+	gain_text = "I feel a shimmer in the air, atmosphere around me gets colder. I feel my body realizing the emptiness of existance. Something's watching me"
+	banned_knowledge = list(/datum/eldritch_knowledge/base_ash,/datum/eldritch_knowledge/base_flesh,/datum/eldritch_knowledge/final/ash_final,/datum/eldritch_knowledge/final/flesh_final,/datum/eldritch_knowledge/base_rust,/datum/eldritch_knowledge/final/rust_final)
+	next_knowledge = list(/datum/eldritch_knowledge/void_grasp)
+	required_atoms = list(/obj/item/kitchen/knife)
+	result_atoms = list(/obj/item/melee/sickly_blade/void)
+	cost = 1
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/base_void/recipe_snowflake_check(list/atoms, loc)
+	. = ..()
+	var/turf/open/turfie = loc
+	if(turfie.GetTemperature() > T0C)
+		return FALSE
+
+/datum/eldritch_knowledge/void_grasp
+	name = "Grasp of Void"
+	desc = "Lowers your victims body temperature by 20 degrees, also temporarily mutes them."
```

Comment:
this should probably mention the muteness first, because it's much, much more significant than decreasing your victim's body temperature by a negligible amount.